pymc/distributions/logprob.py

```diff
@@ -60,7 +63,7 @@ def _get_scaling(total_size, shape, ndim):
     scalar
     """
     if total_size is None:
-        coef = floatX(1)
```

Comment:
Are you sure about not using floatX?

Reply:
was taken care of in the `return at.as_tensor` line